### PR TITLE
chore: clean up synthtool

### PR DESCRIPTION
**Pull request by summer-ji-eng**

### CODE REVIEW

File: synth.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
         'package-name': f'@google-cloud/pubsub',
         'main-service': f'pubsub',
         'bundle-config': f'google/pubsub/{version}/pubsub_gapic.yaml',
-        'template': f'typescript_gapic'
+        'template': f'typescript_gapic',
```
Comment (alexander-fenster) [inline synth.py:23]: nit: here and above, `f'...'` means "formatted string" - the string where you can use `{placeholders}` to insert a variable. It's actually only used in `bundle-config`, but in `package-name`, `main-service`, and `template` there should be no `f'...'`, just regular strings.

File: synth.py [hunk 2]
```diff
@@ -20,7 +20,8 @@
...
+        "iam-service": "true"
```
Comment (alexander-fenster) [inline synth.py:24]: super nit: the script above uses single quotes, not double quotes :)

File: synth.py (hunk 1 above)
Comment (summer-ji-eng) [reply synth.py:23]: gotcha. thanks

**Comment by summer-ji-eng** [comment]
> LGTM only with one question, the `// eslint-disable-next-line @typescript-eslint/no-unused-vars` in test files are auto-generated or manually added? If it's manual work we may want to do it through synth.py to avoid lint error. Thanks!

I guess the test you mention here is manual test not the test generated from micro-generator.

**Pull request merged**